### ISSUE TITLE
Feature request: Option to disable translate on hover

### ISSUE THREAD

**Issue by duongdominhchau** [FilipePS/Traduzir-paginas-web#97]
Recent update added auto translate on hover, but the translation is shown even when the text does not need to be translated. It even translate non-text block and display a black rectangle on the screen. Proper language detection would be harder to add, so I just request for an option to disable the auto-translation.

**Comment by FilipePS** [comment]
In the settings you can disable it.  And for this to not appear on the pages that are in your language, update to the latest version.

**Comment by duongdominhchau** [comment]
Sorry, this time I found it. At the time I open this issue, I only check the **Preferences** through `about:addons`

![image](https://user-images.githubusercontent.com/56427890/110232550-e28dff00-7f50-11eb-93e0-5ad47fb207ee.png)

I rechecked with the extension icon and there are other settings pages.

![image](https://user-images.githubusercontent.com/56427890/110232564-fd607380-7f50-11eb-906e-960175adf958.png)

About the "in my language", I meant my Firefox is set to use English and Vietnamese, with English as the default, but when I'm on a Vietnamese site it still translates. Disabled the auto-translate is enough for me, thank you.
